src/models/user.js

```diff
@@ -0,0 +1,18 @@
+export default Backbone.Model.extend({
+  urlRoot: 'http://tiny-lr.herokuapp.com/collections/users-bbs',
+
+  defaults: {
+    name: 'bbsoucy@gmail.com',
+    password: 'honeycrisp',
+  },
+
+  validate() {
```

Comment:
For this assignment I would recommend renaming `validate` to something like `getFormErrors` or something.

The `validate` function in backbone effects the ability to save things to the server and you want to save all attempts not just successful ones.